app/src/main/res/values/strings.xml

```diff
@@ -35,4 +35,6 @@
     <string name="subtitle_organizer">Organizer: %s</string>
     <string name="event_thumbnail">Event Thumbnail</string>
 
+    <string name="default_base_url">https://eventyay.com/api/v1/</string>
+
```

Comment:
Same reason. But now setting it in code itself to adjust with BuildConfig. As default URL will be different for debug and release mode.